### PR TITLE
Show answer sets even if they are on "step 1" (as answer sets don't usually have more than 1 step)

### LINKED ISSUE #598: When retrieving a list of answer sets, don't limit steps to > 1
After #593, we're no longer correctly loading the list of answer sets because the default is to filter to interview where steps > 1. Answer sets don't have any steps.

We just need to change the way we invoke `get_saved_interview_list` so that `exclude_newly_started_sessions` is False, but we might also consider changing the default to `False` and just using that filter on the `interview_list.yml` page instead of everywhere.

### PR DESCRIPTION
**Pull request by nonprofittechy**
Fix #598

### CODE REVIEW

**Comment by nonprofittechy** [comment]
Note that if you test this in the playground, you'll need to create at least one answer set using the local version of session.py, e.g., by saving an answer set from a run of al_question_test.yml

**Review by BryceStevenWilley — APPROVED** [review]
One small nit, otherwise I also commented on https://github.com/SuffolkLITLab/docassemble-AssemblyLine/issues/598#issuecomment-1297097313, but idk if it's practical or the right way of approaching.

File: docassemble/AssemblyLine/sessions.py [hunk 1]
```diff
@@ -264,7 +264,7 @@ def get_saved_interview_list(
     filename_to_exclude: str = "",
     exclude_current_filename: bool = True,
     exclude_filenames: List[str] = None,
-    exclude_newly_started_sessions: bool = True,
```
[the remark this answers is not in the document]
Comment (nonprofittechy) [reply docassemble/AssemblyLine/sessions.py:267]: False is the default, but the comment is a good suggestion

[the remark this answers is not in the document]
Comment (BryceStevenWilley) [reply docassemble/AssemblyLine/sessions.py:267]: Sorry, I meant True is the better default.

**Pull request merged**